internal/coverprofile/covermerge.go

```diff
@@ -0,0 +1,95 @@
+package coverprofile
+
+// gocovmerge takes the results from multiple `go test -coverprofile` runs and
+// merges them into one profile
+
+// Taken from: https://github.com/wadey/gocovmerge @ b5bfa59ec0adc420475f97f89b58045c721d761c
```

Comment:
I think we may need to copy the license file
(https://github.com/wadey/gocovmerge/blob/master/LICENSE) into `internal/coverprofile` to comply with the license.

Reply:
I added the LICENSE to the file header since it is only the file we copied over. Please let me know if this is good enough.